src/bci_build/package/appcontainers.py

```diff
@@ -724,3 +724,32 @@ def _get_nginx_kwargs(os_version: OsVersion):
     )
     for tomcat_major, os_version in product(_TOMCAT_VERSIONS, ALL_BASE_OS_VERSIONS)
 ]
+
+APACHE2_CONTAINERS = [
+    ApplicationStackContainer(
+        name="apache2",
+        os_version=os_version,
+        package_name="apache2-image",
```

Comment:
We offer various MPM modules, should the name at least reflect the MPM that was chosen?

Reply:
Which name do you suggest? I assumed that the package name would be a good name for the image, but I'm open to alternatives.